src/test/java/webdriver/LogInTest.java

```diff
@@ -0,0 +1,69 @@
+package webdriver;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class LogInTest {
+
+    private WebDriver driver;
+    private static final String USERNAME = "testselenium202208";
+    private static final String PASSWORD = "vMQGe25UrcksBtj";
+    private static final By LOGIN_BUTTON = By.linkText("Log in");
+    private static final By USERNAME_INPUT = By.id("passp-field-login");
+    private static final By USERNAME_SIGNIN_BUTTON = By.xpath("//button[@id='passp:sign-in']");
+    private static final By PASSWORD_INPUT = By.id("passp-field-passwd");
+    private static final By PASSWORD_SIGNIN_BUTTON = By.xpath("//button[@id='passp:sign-in']");
+    private static final By USER_ACCOUNT_NAME = By.className("user-account__name");
+
+    @BeforeEach
+    void startDriver() {
+        WebDriverManager.getInstance(ChromeDriver.class).setup();
+        driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(15, TimeUnit.SECONDS);
```

Comment:
`implicitlyWait(long time, TimeUnit unit);` is deprecated. Please use `implicitlyWait(Duration.ofSeconds(15)`